app/assets/javascripts/metrics.js

```diff
@@ -156,3 +162,70 @@ function displayWeeklyBreakdown(where, rawDataSource) {
     
     chart.draw(processedData);
 }
+
+function thisYear() {
+    return parseInt(d3.select("#year").html());
+}
+
+function displayPerformanceLineChart(where, performanceData, volumeData) {
+    var xRange = [new Date(thisYear() - 1, 11, 29), new Date(thisYear() + 1, 0, 1)];
```

Comment:
I see. Is there some way you can make this more obvious in the code, so that somebody doesn't try to correct it to '31' in the future?